client/src/App.tsx

```diff
@@ -25,8 +37,30 @@ function App() {
             <Route path="/" element={<Greetings />} />
             <Route path="/signup" element={<Signup />} />
             <Route path="/login" element={<Login />} />
-            <Route path="/home" element={<Home />} />
-            <Route path="/user-settings" element={<UserSettings />} />
+            <Route
+              path="/home"
+              element={
+                <RequireAuth>
+                  <Home />
+                </RequireAuth>
+              }
+            />
+            {/* <Route
+              path="/expenses"
+              element={
+                <RequireAuth>
+                  <Expenses />
+                </RequireAuth>
+              }
+            /> */}
```

Comment:
Remove commented code

Reply:
I will uncomment before merging. this branch does not have the Expenses page yet. will merge once Expenses is merged